samples/benchmarks/coremark/README.rst

```diff
@@ -0,0 +1,146 @@
+.. _coremark_sample:
+
+CoreMark
+########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to run the `CoreMark®`_ benchmark to evaluate performance of a core.
```

Comment:
Can the sample be renamed to a name that does not just include a registered trademark name? For similar reason for which we don't name any sample just "Bluetooth".
Suggestions: `CoreMark benchmark evaluation` or `CoreMark evaluation`

Reply:
Don't understand ... 
We are literally using source of coremark here so why cannot we use this name? Is there some legal problem with using this name?

Reply:
Just the name of the sample from CoreMark to CoreMark benchmark evaluation (or without the word benchmark).

Reply:
Could you explain to me why it should be changed?

Reply:
Just to clarify directly in the sample name what the CoreMark is used for. I agree it could also be left as is (something similar would be for example Memfault).

Reply:
If that the only cause then name Coremark is for me much more clear then "CoreMark evaluation". We are running sample from coremark repository https://github.com/eembc/coremark
The part that is in NCS is just porting layer and some stuff like buttons and leds.

Reply:
Calling a sample with a pure trademark name will be problematic if there are more pages about CoreMark appearing anywhere in the docs.

Reply:
@zycz , question: Are you using the entirety of the benchmark suite or just a subset of its features?

Regardless of the answer, @peknis mentioned Memfault. If this sample is integrating CoreMark repository into the NCS, the documentation also requires an integration page: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/integrations.html

Template here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/Integration_template_README.html

Reply:
Clarified with @zycz that this sample integrates all of the benchmark suite. I'm fine with following the Memfault case then, as long as there is the integration doc added.